docs/topics/proc_attaching-a-remote-debugger-to-the-application.adoc

```diff
@@ -29,5 +29,10 @@ To quickly launch your debug configuration after the first time, click *Run->Deb
 
 .Additional resources
 
-* link:https://access.redhat.com/articles/1290703[Debug an OpenShift Java Application with JBoss Developer Studio] on Red Hat Knowledgebase.
+* link:https://access.redhat.com/articles/1290703[Debug an OpenShift Java Application with {DevStudio}] on Red Hat Knowledgebase.
```

Comment:
I thought we agreed on not replacing this occurrence, or am I mistaken?

Reply:
I wanted to preview how it would look like if we do, as mentioned in JIRA comment. Doesn't look good, does it? As this is a separate commit from other changes, I can revert this particular change without affecting anything else, as soon as I get a go ahead. WDYT?

Reply:
commit to revert: 40228c56a06d99a5dc43059df01bb1f3646699ef

Reply:
I would keep the name of the kbase article, and I would put the information about the original name in normal text. I don't think an admonition is warranted here.